machines/request.ts

```diff
@@ -54,6 +54,7 @@ const model = createModel(
       ACCEPT: () => ({}),
       ACCEPT_AND_VERIFY: () => ({}),
       REJECT: () => ({}),
+      GO_TO_RECEIVED_VC_TAB: () => ({}),
```

Comment:
Is this required?

Reply:
Yes, for the typing. Without this, State machines types will show error when we catch the event in `on` section